k8s/production/custom/rotate-keys/cron-jobs.yaml

```diff
@@ -23,6 +23,8 @@ spec:
                 secretKeyRef:
                   name: rotate-keys
                   key: gitlab-token
+            - name: SKIP_GITLAB_VARIABLE_UPDATE
+              value: "true"
```

Comment:
Does kube automatically convert this to a boolean type so the check of the environment variable will work as written?

Reply:
I'm not sure, but `"true"` is truthy regardless.